content/en/docs/languages/node/quickstart.md

```diff
@@ -18,7 +18,7 @@ and other tutorials):
 
 ```sh
 # Clone the repository to get the example code
-$ git clone -b {{< param grpc_vers.core >}} --depth 1 https://github.com/grpc/grpc
+$ git clone -b {{< param grpc_vers.core >}} --depth 1 --shallow-submodules https://github.com/grpc/grpc
```

Comment:
Tangential to the contents of this PR, but do the docs really still recommend the core-based Node implementation? I don't think this is up-to-date. @murgatroid99 Can you please advise?

Reply:
I vaguely recall that it's because the Node pages didn't get updated (ran out of time) -- but I could be misremembering. I'll let @murgatroid99 answer.